Pbi 20 create assessment event

whenwe177 opened the pull request:
1. Created new get_assessment_tool_by_company function.

Method signature: (user: User) => AssessmentTool[]
Returns the list of assessment tools belonging to the user's associated company.
Throws error if user is not an assessor.

Endpoint protected with IsAuthenticated decorator.

whenwe177 committed:
[RED] create tests for getting list of tools endpoint

Tests for getting the list of tools using the newly created get_assessment_tool_by_company function

whenwe177 committed:
[GREEN] implement getting list of assessment tools

Only assessors are able to get the list of assessment tools specific to the assessor's bounded company. The API route is protected with authorisation.

whenwe177 committed:
[REFACTOR] add a 'type' attribute on the value returned by the assessment tool endpoint

johaneschristian commented on assessment/views.py:
```diff
@@ -4,15 +4,28 @@
+@require_GET
+@api_view(["GET"])
+@permission_classes([IsAuthenticated])
+def get_assessment_tool(request):
```
Hello, Alex. To follow our convention on the views functions, is it possible to rename this function to serve_get_assessment_tool_by_company instead? Thanks!

whenwe177 replied:
Hello, Jo. Thanks for the suggestions. Will do them in the next commit.

whenwe177 committed:
[REFACTOR] remove unneccessarry spaces, rename function name

Rename function name from get_assessment_tool to serve_get_assessment_tool to be more semantic, also remove unneccessary trailing spaces

whenwe177 committed:
[CHORES] merge incoming changes and modify .gitignore

Merge the changes from previous commits, modify gitignore to allow pycache not to be comitted

whenwe177 committed:
[REFACTOR] refactor function from views to service

Refactor serializeDataUsingSerializer function, initially in views.py, move it to services/assessment_tool.py

whenwe177 committed:
[REFACTOR] rename return type from data to serialized_assignment_list
